integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/cleanfiles/TestCleanFilesCommandPartitionTable.scala

```diff
@@ -186,7 +193,9 @@ class TestCleanFilesCommandPartitionTable extends QueryTest with BeforeAndAfterA
     removeSegmentEntryFromTableStatusFile(CarbonEnv.getCarbonTable(Some("default"), "cleantest")(
       sqlContext.sparkSession), "2")
 
-    sql(s"CLEAN FILES FOR TABLE CLEANTEST").show()
+    val df1 = sql(s"CLEAN FILES FOR TABLE CLEANTEST DRYRUN")
```

Comment:
please verify (at least manully) that dry run size and without dry run, it shows the correct size estimation (if somewhere we are adding from two places, it might give incorrect results)

Also verify the summary is same as actual size cleaned up with big table in backend for both partition and non-partition table.

Reply:
added code to check the segment size in one of the test case and checking it with clean files result and the dry run result. The test case will only pass when all three result are the same